code/game/objects/structures/guillotine.dm

```diff
@@ -147,36 +147,34 @@
 
 		var/obj/item/bodypart/head/head = H.get_bodypart("head")
 
-		if (QDELETED(head))
-			return
-
 		playsound(src, drop_sound, 100, TRUE)
 		if (blade_sharpness >= GUILLOTINE_DECAP_MIN_SHARP || head.brute_dam >= 100)
-			head.dismember()
-			log_combat(user, H, "beheaded", src)
-			H.regenerate_icons()
-			unbuckle_all_mobs()
-			kill_count += 1
-
-			var/blood_overlay = "bloody"
-
-			if (kill_count == 2)
-				blood_overlay = "bloodier"
-			else if (kill_count > 2)
-				blood_overlay = "bloodiest"
-
-			blood_overlay = "guillotine_" + blood_overlay + "_overlay"
-			cut_overlays()
-			add_overlay(mutable_appearance(icon, blood_overlay))
-
-			// The crowd is pleased
-			// The delay is to making large crowds have a longer laster applause
-			var/delay_offset = 0
-			for(var/mob/M in viewers(src, 7))
-				var/mob/living/carbon/human/C = M
-				if (ishuman(M))
-					addtimer(CALLBACK(C, TYPE_PROC_REF(/mob/, emote), "clap"), delay_offset * 0.3)
-					delay_offset++
+			if(head)
```

Comment:
turn this into an early return, like

```suggestion
			if(QDELETED(head))
				return
			// code goes here	
```

Reply:
No wait never mind im stupid

Reply:
mhm, no idea how to make it work through return